ml-agents/setup.py

```diff
@@ -76,6 +76,7 @@ def run(self):
         "cattrs<1.1.0; python_version<'3.8'",
         "cattrs>=1.1.0,<1.7; python_version>='3.8'",
         "attrs>=19.3.0",
+        "huggingface_hub~=0.8",
```

Comment:
```suggestion
        "huggingface_hub>=0.12",
```

I would suggest a more recent version, just in case (latest is currently 0.14.1)